raft-tech/cfgov-refresh
# revised header details on fix-it strategies screen

Client wanted several changes to the header of the Fix-it Strategies.  This pertains to #206 in the Money-Tools project board.


## Changes

- adjusted formatting of headers in strategies.less file
- moved headers in the /strategies/fix-it/index.js file


## How to test this PR

1.  Open and clear application cache.
2.  Complete the _Money-on-hand_ screens (do not need starting balance).
3.  Once on the _Calendar View_, add an expense such that you get a _red_ week.
4. Tap the red week and go to the _Fix-It_ screen.
5. Note that the _week header_, the change in location of the _Fix-it header_ and the removal of the _General Strategies header_ and accompanying copy.  It should look like the screenshot below.


## Screenshots

<img width="200" alt="Screen Shot 2020-08-12 at 4 23 20 PM" src="https://user-images.githubusercontent.com/34319929/90064125-4f416380-dcb8-11ea-9c44-bd6cc57aeff2.png">
